Added support for two new boards: LILYGO T-Display-S3-Pro-MVSRLora and LILYGO T-Display-S3-Pro-MVSRLora_NO_BATTERY.

Llgok opened the pull request:
The product images are as follows:
![IMG_20250424_143615](https://github.com/user-attachments/assets/055b4e43-cd55-400d-8b85-7f2a4a144733)
![IMG_20250424_150605](https://github.com/user-attachments/assets/f008591b-85eb-4bca-8abc-1628d6ad9b58)

The compiled firmware version 1.6.1 is as follows:
[[T-Display-S3-Pro-MVSRLora_V1.0][xiaozhi_V1.6.1]_firmware_202504291103](https://github.com/Xinyuan-LilyGO/T-Display-S3-Pro-MVSRLora/blob/main/firmware/%5BT-Display-S3-Pro-MVSRLora_V1.0%5D%5Bxiaozhi_V1.6.1%5D_firmware_202504291103.bin)
[[T-Display-S3-Pro-MVSRLora_No_Battery_V1.0][xiaozhi_V1.6.1]_firmware_202504291103](https://github.com/Xinyuan-LilyGO/T-Display-S3-Pro-MVSRLora/blob/main/firmware/%5BT-Display-S3-Pro-MVSRLora_No_Battery_V1.0%5D%5Bxiaozhi_V1.6.1%5D_firmware_202504291103.bin)

78 commented:
1. Please segregate the bug fix code and the new boards code into two separate pull requests.
2. The root README.md file does not have sufficient space to accommodate information about new boards. Consequently, please avoid making any modifications to this file.

Llgok commented:
@78 Hello, I have changed the pull request to add support for two boards: LILYGO T-Display-S3-Pro-MVSRLora and LILYGO T-Display-S3-Pro-MVSRLora_NO_BATTERY, and also removed the modification to the README.md file in the root directory.

78 commented:
No PMIC or battery gauge?

Llgok commented:
@78 The code Sy6970 in the text refers to the PMIC.

78 commented:
I've double checked your product images. Found colors of emotional emojis might be incorrect.

Llgok commented:
@78 This was indeed an error; however, I have already fixed it. In `panel_config`, the member `rgb_ele_order` needs to be set to `LCD_RGB_ELEMENT_ORDER_BGR`.
![1](https://github.com/user-attachments/assets/0b637386-559a-46a4-b56a-810cb850dd31)
